src/main/java/seedu/address/logic/parser/AddEventCommandParser.java

```diff
@@ -33,12 +33,15 @@ public AddEventCommand parse(String args) throws ParseException {
         }
 
         EventName eventName = ParserUtil.parseEventName(argMultimap.getValue(PREFIX_EVENT_NAME).get());
-        DateTime startDatetime = ParserUtil.parseDateTime(argMultimap.getValue(PREFIX_START_DATE_TIME).get());
-        DateTime endDatetime = ParserUtil.parseDateTime(argMultimap.getValue(PREFIX_END_DATE_TIME).get());
-        if (!DateTime.isValidDateRange(startDatetime.toString(), endDatetime.toString())) {
+        DateTime startDateTime = ParserUtil.parseDateTime(argMultimap.getValue(PREFIX_START_DATE_TIME).get());
+        DateTime endDateTime = ParserUtil.parseDateTime(argMultimap.getValue(PREFIX_END_DATE_TIME).get());
+
+        assert startDateTime != null && endDateTime != null : "Start and end date times of an event must not be null";
+
+        if (!DateTime.isValidDateRange(startDateTime.toString(), endDateTime.toString())) {
             throw new ParseException(DateTime.MESSAGE_CONSTRAINTS);
```

Comment:
I went to check the code coverage problem, not sure if you want but I think line 42 the exception throwing is not covered, so perhaps could add a small parse_invalidDateTimeRange_failure test that throws the exception in addcommandparsertest

Reply:
Thanks for checking! I have added the test 😄